course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,103 @@
+| 4    | Sep 19 | Intro to dplyr                                      | Joel           |
+| 5    | Sep 21 | More dplyr                                          | Joel           |
+| 6    | Sep 26 | Qualitative review of population models             | Madeleine      |
+| 7    | Sep 28 | Qualitative population models in 2 dimensions       | Madeleine      |
+| 8    | Oct 03 | Fixed points, null clines, and stability            | Madeleine      |
+| 9    | Oct 05 | Modeling in R                                       | Madeleine      |
+| 10   | Oct 10 | Fitting models to data                              | Madeleine      |
+| 11   | Oct 12 | Linear regression population dynamics models?       | Luke           |
+| 12   | Oct 17 | Introduce data sets, hypotheses, start projects     | Luke           |
+| 13   | Oct 19 | Project work, prodigenR, tidy data, git via rstudio | Luke           |
+| 14   | Oct 24 | Data visualization in R                             | Lindsay        |
+| 15   | Oct 26 | Project work, data visualizations                   | Lindsay        |
+| 16   | Oct 31 | Reproducible workflow, metadata, Rmarkdown          | Lina           |
+| 17   | Nov 02 | Project work, metadata                              | 1 TA/group     |
+| -    | Nov 07 | Fall break                                          | -              |
+| -    | Nov 09 | Fall break                                          | -              |
+| 18   | Nov 14 | Scientific collaboration, Git and GitHub            | Elliot         |
+| 19   | Nov 16 | Project work                                        | 1 TA/group     |
+| 20   | Nov 21 | Preparing a scientific item for academic publishing | Lina           |
+| 21   | Nov 23 | Project work                                        | 1 TA/group     |
+| 22   | Nov 28 | Project work                                        | 1 TA/group     |
+| 23   | Nov 30 | Project work                                        | Everyone       |
+| 24   | Dec 05 | Project work                                        | Everyone       |
+
+# Assessment schedule 
+| Assesment             | Type                | Due date   | Marks |
+|-----------------------|---------------------|------------|-------|
+| Getting set up        | Individual          | Sep 11     | 4     |
+| Basic R               | Individual          | Sep 18     | 8     |
+| dplyr                 | Individual          | Sep 25     | 8     |
+| Population models     | Individual          | Oct 02     | 8     |
+| Modeling in R         | Individual          | Oct 09     | 8     |
+| Linear regression?    | Individual          | Oct 16     | 8     |
+| Setup github          | Project, Individual | Oct 23     | 8     |
+| Data visualization    | Project, Individual | Oct 30     | 8     |
+| Reproducible workflow | Project, Individual | Nov 06     | 8     |
+| Final project work    | Project, Group      | Dec 9 – 20 | 32    |
```

Comment:
We should decide on the date for the final assignment. The range above is from the schedule Luke sent regarding when finals are taking place. When are we due to grade the final assignment? I will be gone Dec 14th - Jan 5th.